evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -0,0 +1,38 @@
+from jina.executors.evaluators import BaseEvaluator
+from typing import Sequence, Any
+import nltk.translate.bleu_score as bleu
```

Comment:
import these dependencies in the evaluate method

Reply:
Try following standard when importing.

1. System lib dependencies
Space
2. External imports
Space
3. Imports from jina